addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14334,7 +14337,7 @@ msgstr ""
 #. Description of setting "Videos -> Playback -> Allow hardware acceleration (VDPAU)" with label #13425
 #: system/settings/settings.xml
 msgctxt "#36155"
-msgid "Enable VDPAU hardware decoding of video files, mainly used for NVIDIA graphics and in some circumstances AMD graphics."
+msgid "Enable VDPAU hardware decoding of video files, mainly used for Nvidia graphics and in some circumstances AMD graphics."
```

Comment:
Well, there is a history in Transifex, but it seems it gets lost in our "export to Transifex" process. 

http://docs.transifex.com/faq/#what-happens-when-i-update-my-source-files is reading just fine, but imo it does not work (at least for us).